### ISSUE TITLE
Item sale prices are weird (player selling to merchant)

### ISSUE THREAD

**Issue by mkahvi** [SvenWerlen/fvtt-loot-sheet-npc-pf1#4]
```
count | item         | per item | sold at | expected sale | excess profit
----------------------------------------------------------|---------------
   9  | short spear  | 1 gp     | 9 gp    | 4.5 gp        | +100%
   4  | longspear    | 5 gp     | 12 gp   | 10 gp         | +20%
   1  | heavy flail  | 15 gp    | 8 gp    | 7.5 gp        | +7%
```

Overall it looks like numbers are being rounded up to nearest gold for some reason, except the case with the longspear stack where it just gave really strange sum.

Shortbow, chainshirt, and some other items that divide by 2 into neat even gold value seem to be fine.

**Comment by ArdonDaros** [comment]
An option to set Sell Price modifier (Player to Merchant) would be nice as well. This would allow trade goods, such as gems and gold / silver bars, be used as a currency worth their face value.

**Comment by mkahvi** [comment]
> An option to set Sell Price modifier (Player to Merchant) would be nice as well. This would allow trade goods, such as gems and gold / silver bars, be used as a currency worth their face value.

Trade goods are sold at 100% value already from the look of the code, which is rules correct with PF1.

**Comment by SvenWerlen** [comment]
The 3 test cases have been tested successfully locally. The fix will be available in the next version today.

**Comment by SvenWerlen** [comment]
Available in `v-1.8.0`.
